### PR TITLE
Rename block type from questionnaire to question

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Changing of Questionnaire to Question now in blocks

### How to review 
Review this first: https://github.com/ONSdigital/eq-schema-validator/pull/14
Check all instances of Questionnaire have been changed and everything still functions i.e. tests etc.

### CODE REVIEW

**Review by shahi645 — CHANGES_REQUESTED** [review]
You need to update the template file names, to `question.html`

**Comment by shahi645** [comment]
@MIslam92 Will we need to update mbs and e-commerce also ?

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
There are a number of places in the code that rely on the block type being `'Questionnaire'` that need to be updated e.g. https://github.com/ONSdigital/eq-survey-runner/blob/master/app/views/questionnaire.py#L582

File: app/views/questionnaire.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 logger = get_logger()
 
-questionnaire_blueprint = Blueprint(name='questionnaire',
+questionnaire_blueprint = Blueprint(name='Question',
```
Comment (ajmaddaford) [inline app/views/questionnaire.py]: This shouldn't have changed name

Comment (shahi645) [reply app/views/questionnaire.py]: with this change, should be good to go !

**Review by shahi645 — APPROVED** [review]
Good to go, tested against:
https://github.com/ONSdigital/eq-schema-validator/pull/14
Will need to be merged first

**Comment by jonnyshaw89** [comment]
@MIslam92 `mbs_0111.json` and `e_commerce.json` have not been updated

**Comment by jonnyshaw89** [comment]
@MIslam92 you should probably rebase too to resolve the conflicts

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]
you have an empty file `app/helpers/schema_helper.py` possibly a merge issue?

**Review by jonnyshaw89 — COMMENTED** [review]
You also still have this file `tests/app/helpers/test_schema_helper.py`

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**